### PR TITLE
Convert html_head.jsp template to handlebars 

### PR DESCRIPTION
**Pull request by PaulMorris**
This PR affects the contents of the head tag for provider accounts.  CSS and JS files are loaded and working correctly and the title is set correctly for all the 'provider' pages.

To test this PR, log in to the p1 (provider) account and click through the provider pages (like provider/dashboard/, provider/enrollment/, provider/profile, etc.).  They should appear as usual (the CSS files should be loaded and styling the pages) and should be working as normal (JS files loaded and working).  The selenium tests should pass.

### CODE REVIEW

**Comment by PaulMorris** [comment]
If you like you could check the page source.  Here's the old <head> section:
```
    <head>
        

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" href="/cms/css/reset.css" type="text/css" media="all"/>
<link rel="stylesheet" href="/cms/css/style.css" type="text/css" media="all"/>
<link rel="stylesheet" href="/cms/css/jquery.ui.css" type="text/css" media="all"/>

<!--[if IE 7]>
<link rel="stylesheet" href="/cms/css/style-ie7.css" type="text/css" />
<![endif]-->
<script src="/cms/js/jquery-1.7.1.min.js" type="text/javascript"></script>
<script type="text/javascript" src="/cms/js/jquery.tablesorter.min.js"></script>
<script type="text/javascript" src="/cms/js/jquery.tablesorter.widgets.js"></script>
<script src="/cms/js/jquery.ui.core.js" type="text/javascript"></script>
<script src="/cms/js/jquery.ui.widget.js" type="text/javascript"></script>
<script src="/cms/js/jquery.ui.datepicker.js" type="text/javascript"></script>
<script src="/cms/js/jquery.maskedinput.min.js" type="text/javascript"></script>
<script src="/cms/js/jquery.compare.js" type="text/javascript"></script>
<script src="/cms/js/script.js" type="text/javascript"></script>

        <title>Login</title>
    </head>
```
And the new one:
```
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

  <title>Login</title>
  <link rel="stylesheet" href="/cms/css/reset.css" type="text/css" media="all" />
  <link rel="stylesheet" href="/cms/css/style.css" type="text/css" media="all" />
  <link rel="stylesheet" href="/cms/css/jquery.ui.css" type="text/css" media="all" />

  <script src="/cms/js/jquery-1.7.1.min.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.tablesorter.min.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.tablesorter.widgets.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.ui.core.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.ui.widget.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.ui.datepicker.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.maskedinput.min.js" type="text/javascript"></script>
  <script src="/cms/js/jquery.compare.js" type="text/javascript"></script>
  <script src="/cms/js/script.js" type="text/javascript"></script>
</head>
```

File: psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp [hunk 1]
```diff
@@ -7,148 +7,148 @@
...
-	                                    </div>
-	                                </div>
-	                            </div>
-	                            <!-- /.section -->
-	                            <div class="tl"></div>
-	                            <div class="tr"></div>
-	                            <div class="bl"></div>
-	                            <div class="br"></div>
-	                        </div>
+    <html xmlns="http://www.w3.org/1999/xhtml">
```
Comment (jasonaowen) [inline psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp]: It looks like you're re-indenting with 4-space indents here, while you have 2-space indents in your handlebars template. We'd previously talked about using 2-space indents in HTML-ish things generally; would you mind reformatting?

Comment (PaulMorris) [reply psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp]: OK, will do.  (For these jsp files I noticed mixed spaces and tabs, so I went ahead and fixed that, converting to spaces.  I thought about re-indenting but then thought I'd leave that for the conversion to handlebars phase.  But we might as well re-indent while fixing white space.)

Comment (jasonaowen) [reply psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp]: :+1: Thanks!

**Comment by PaulMorris** [comment]
Now with two-space indentation in all affected JSP files.

File: psm-app/cms-web/WebContent/WEB-INF/pages/accounts/register.jsp [hunk 2]
```diff
@@ -2,106 +2,137 @@
...
+                  </div>
+                </div>
+                <!-- /.section -->
+                <div class="tl"></div>
+                <div class="tr"></div>
+                <div class="bl"></div>
+                <div class="br"></div>
+              </div>
+              <div class="buttonBox">
+                <a href="<c:url value=" /login" />" class="greyBtn">
```
Comment (jasonaowen) [inline psm-app/cms-web/WebContent/WEB-INF/pages/accounts/register.jsp]: It looks like this changed added some unwanted whitespace.

Comment (PaulMorris) [reply psm-app/cms-web/WebContent/WEB-INF/pages/accounts/register.jsp]: Good catch.  Seems this was caused by the HTML auto-formatter getting confused by the JSP tag format.

**Pull request merged**